src/py/aspen/database/models/sequences.py

```diff
@@ -180,6 +180,11 @@ class PathogenGenome(Entity):
         ),
     )
 
+    pangolin_lineage = Column(String)
+    pangolin_probability = Column(Integer)
+    pangolin_version = Column(String)
+    pangolin_last_updated = Column(DateTime)
+
```

Comment:
I've been marking `nullable=True`, even though it is the default, to make it super clear what the schema is.